assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -134,9 +143,36 @@ const List = styled(Area).attrs({
   }
 `;
 
-const Li = styled.li.attrs({
-  role: 'option',
-})`
+const PageSeparator = styled(ReorderableSeparator)`
+  position: absolute;
+  bottom: 0;
+  left: ${({ width }) => width / 2}px;
+  width: ${({ width, margin }) => width + margin}px;
+  height: ${({ height }) => height - THUMB_FRAME_HEIGHT}px;
+  display: flex;
+  justify-content: center;
+
+  &:first-of-type {
+    left: -${({ width, margin }) => (width + 2 * margin) / 2}px;
+  }
+
+  & > div {
```

Comment:
I know we don't allow a way to style the `Line` inside the separator here as it is, but I'd definitely prefer a better way than this. We could pass in `lineStyle={css``}`. Or it could be a render prop as `renderLine={(Line) => styled(Line)``}` defaulting to identity function.

Reply:
Made it into a child